Framework/Types/inc/MantidTypes/TofEvent.h

```diff
@@ -0,0 +1,119 @@
+#ifndef MANTID_TYPES_TOFEVENT_H
+#define MANTID_TYPES_TOFEVENT_H
+
+#include "MantidTypes/DateAndTime.h"
+
+namespace Mantid {
+namespace DataObjects {
+class EventList;
+class WeightedEvent;
+class WeightedEventNoTime;
+class tofGreaterOrEqual;
+class tofGreater;
```

Comment:
Not all of these seem necessary (or even exist?)?

Reply:
You're right, I didn't notice that.